### ISSUE TITLE
stk1160 PAL only black/white /NTSC working // with NTCS random 0.3sec black LED

### ISSUE THREAD

**Issue by lobocobra** [tvdzwan/hyperion#200]
Hi all

I currently try to figure out, if I do have a HW driver problem with my EasyCap 1160 OR a Hyperion SW Problem.

Problem description (2 issues)
1) if I set my HDMI2AV converter on PAL, my LED only reflect a Black/White picture. A screenshot shows, that in deed, only a bad quality b/w picture is handled. With NTSC I do get better quality and LED show the right colors (I checked in Windows all HW, so no HW is defect).
=> current work-around: I set HDMI2AV converter on NTSC, then it works
2) depending on the BAUD I put in the hyperion.conf.json file, my LEDs go all together black for a blink of an eye randomly all 3-40sec. (if I use hyperion-remote -c xxx, the color are steady with no interruption)
=> current work-around: I must set the BAUD value to 100000 as a result, I have the LED only all 40-60 sec going black (all LED together for 0.3 sec)

My questions:
1) does anybody else have this random shut - down of all LED and having them going black for a blink of an eye?
=> if I change the BAUD settings in the config file, I can influence the time between all LED going BLACK. With 100000 Baud I get best result (40sec -2min) , if I go for 1000000, then I have it like all 2 sec like a flicker of all LED, as they go very shortly black (very distrubing). !!! my LED do NOT Flicker and I have no issue with grounding or shielding. Hyperion-remote -c blue, works for hours without flickering.
2) does anybody have a solution for the HDM2AV problem? why should I not use PAL in Europe? If v412-ctl -all shows  "Video Standard = 0x0000100	NTSC-M". I assume the problem that the SDK1160 is working in NTSC mode, while a PAL signal is incoming.

My config
v412-ctl --all: http://pastebin.com/TqATkQz9
hyperion config file: http://pastebin.com/mR6X9yKP (I did not past the 99 LED, as the config works)
=> changing the baud value seems to help, any other variable has no impact.

My HW:
- RaspPi standard installation v7.6 (all works well), latest updates
- Kernel: raspcat 3.12.28+
- HDMI 4x splitter with sep power 
- LED stripe with 100 LED lpd8806
- recommended HDMI2AV converter (with shared power)
- EasyCap SDK1160
- 3.5 Amp current (all LED are working well no flackering)

**Comment by lobocobra** [comment]
In order to narrow the issue, I wanted to eliminate HW errors. So I installed ambi-tv and made 2 interesting observations:
- the PAL/NTSC issue is completely gone using latest ambi-tv with V4l2
  => it seems to be an issue in the hyperion code
- the flickering is almost gone, if I set the buffer at 3, but worse with any other buffer settings
  => NTSC seems here to be worse than PAL, thus it could have to do with frames

I keep on testing...

**Comment by lobocobra** [comment]
Problem solved.
Somehow I achieved to solve the issue, but could not really narrow it down. Here what I did:
- re-installed raspi from scratch, whith the latest version and updated linux to its latest (v7.8)
- I did replace the EasyCap by a new HW (same bios) and after some trying I manage now to run the HDMI converter in PAL mode
- as a last thing I did loop the + / - 5V so that the start and the end of the stripe is connected with 5V (did probably not help)
- I have now a stronger power supply but this did not help (as the old one workes now also)
- Gamma Value is now at 1 instead of 2 (did for sure not cause it)
For me the issue seems to be fully solved and I do hope, that it helps somebody else.
